frontend/packages/ux-editor/src/containers/DesignView/AddItem/FeedbackForm.tsx

```diff
@@ -0,0 +1,58 @@
+import type React from 'react';
+import { submitFeedbackPath } from 'app-shared/api/paths';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { post } from 'app-shared/utils/networking';
+import { FeedbackFormImpl } from '@studio/feedback-form';
+import { toast } from 'react-toastify';
+
+export function FeedbackForm(): React.ReactNode {
+  const { org, app } = useStudioEnvironmentParams();
+
+  const submitFeedback = async (answers: Record<string, string>) => {
+    // Using explicit texts here to avoid adding these potentially
+    // temporary and unnecessary translations to the translation files.
+    try {
+      // Using regular axios post rather than a mutation hook, since we are not storing
+      // the feedback in the cache, nor are we updating any state.
+      await post(submitFeedbackPath(org, app), { answers: { ...answers } });
+      toast.success('Takk for tilbakemeldingen!');
```

Comment:
Should this be in nb.json?🤔

Reply:
I added a comment 4 lines above this stating that we are using explicit texts 😄